server.go

```diff
@@ -215,7 +215,7 @@ type Server struct {
 	WriteTimeout time.Duration
 
 	// IdleTimeout is the maximum amount of time to wait for the
-	// next request when keep-alives are enabled. If IdleTimeout
+	// next request when keep-alive are enabled. If IdleTimeout
```

Comment:
If you're going to fix it then please change it to `next request when keep-alive is enabled.`. Thanks.

Reply:
Done.